packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -276,6 +271,18 @@ abstract class FlutterCommand extends Command<void> {
       help: 'The URL to provide to the browser. Defaults to an HTTP URL with the host '
           'name of "--web-hostname", the port of "--web-port", and the path set to "/".',
     );
+    addWebBrowserFlags(hide: !verboseHelp);
+  }
+
+  void addWebBrowserFlags({required bool hide}) {
+    argParser.addOption(
+      FlutterOptions.kWebBrowserFlags,
+      help: 'Additional flags to pass to a browser instance at startup.\n'
+          'Chrome: https://peter.sh/experiments/chromium-command-line-switches/\n'
+          'Firefox: https://wiki.mozilla.org/Firefox/CommandLineOptions\n',
```

Comment:
I think we can cut the firefox link, flutter run doesn't work with it.

For the chrome link, do we have something more authoritative ?

Reply:
Ok, we can cut it for firefox :)
I searched the whole web, even the official Chromium download page links to it: https://www.chromium.org/developers/how-tos/run-chromium-with-flags/

Reply:
I would link to https://www.chromium.org/developers/how-tos/run-chromium-with-flags/ then